modules/ROOT/pages/depl-examples/ubuntu-compose/ubuntu-compose-prod.adoc

```diff
@@ -0,0 +1,387 @@
+:description: Install Infinite Scale using Docker Compose on a server for production use.
+
+// the folder to use for the example
+:ocis_wopi: ocis_wopi
+
+include::partial$multi-location/compose-version.adoc[]
+
+image:depl-examples/ubuntu-compose/ubuntu-basic-teaser-image.png[Teaser Image, width=650]
+
+{empty} +
+
+toc::[]
+
+== Introduction
+
+{description} The aim of this guide is to be up and running as fast as possible using a deployment setup that includes *Infinite Scale and web office applications for document collaboration* for home usage or small businesses. It also uses valid certificates from Letsencrypt.
+
+NOTE: This guide references the latest production version of Infinite Scale.
+
+== Requirements
+
+=== Hardware
+
+This guide describes an installation of Infinite Scale based on Ubuntu LTS and docker compose. The underlying hardware of the server can be anything as listed below as long it meets the OS requirements defined in the xref:software-stack[Software Stack]:
+
+* Raspberry Pi (4 and higher)
+* Bare Metal Server
+* Virtual Machine
+* ...
+
+[NOTE]
+====
+* Disk space +
+About 2.8GB of disk space is needed as you not only get Infinite Scale but also office packages for online collaboration and other required software to run this setup. 
+
+* Memory +
+We recommend at minimum 4-6GB of memory. 
+====
+
+=== Knowledge Stack
+
+You, as administrator, must have the following minimum knowledge stack.
+
+* Being capable to order and configure external accessible domains.
+
+* Being capable to configure: +
+Router, Firewall, NAT, DHCP, networks ect.
+
+* Command line (bash) tools like: +
+ssh, actions on files, edit files etc.
+
+* Maintaining a server: +
+Setting or changing hostnames, IP configuration, installing SW packages etc.
+
+=== Software Stack
+
+For the OS, *Ubuntu LTS 24.04* has been selected, but it will also work with Ubuntu LTS 22.04. If you already have a server running Ubuntu LTS 24.04, you can use that one as long it meets the requirements listed below. 
+
+Note that this guide expects:
+
+* The prerequisite of a server with the installed OS is met and any required software other than Infinite Scale like docker and docker compose is installed and preconfigured and all the software is updated to the latest version.
+
+* You have at minimum shell access and you can reach your host via ssh from another computer in your network. `sudo` permissions may be required in some steps later in the setup.
+
+=== Other Requirements
+
+Firewall::
+Because this server is exposed to the internet, we highly recommend using a firewall for security reasons. This can be an external firewall or a firewall configured on the server. The minimum configuration for the firewall is to allow port 443. Note that this document *does not* cover configuring any kind of firewall.
+
+Domain Name and Routing::
+To access Infinite Scale from the internet, you *must*:
++
+--
+* Own a domain name which will get multiple subdomains configured.
```

Comment:
```suggestion
* Own a domain name which will get multiple subdomains configured.
** You can also use a wildcard operator for the subdomains in the DNS configuration. In case where you already use subdomains for other purposes, we recommend using a subdomain as entry point and add the wildcard operator there to allow different IP configurations.
```